entc/integration/integration_test.go

```diff
@@ -2870,6 +2871,31 @@ func OrderByEdgeTerms(t *testing.T, client *ent.Client) {
 	}
 }
 
+func InsertBulkM2MAdd(t *testing.T, client *ent.Client) {
+	require := require.New(t)
+	ctx := context.Background()
+
+	steve := client.Student.Create().SetName("Steve").SaveX(ctx)
+	bob := client.Student.Create().SetName("Bob").SaveX(ctx)
+	will := client.Student.Create().SetName("Will").SaveX(ctx)
+
+	math := client.Subject.Create().SetName("Math").
+		AddStudents(steve, bob).
```

Comment:
@a8m the main issue was, that when adding multiple students to a M2M edge which have a through table, and the through table has a default id, then the insertion fails because all of the ids were reused

Reply:
Without this fix you will get for this test:

panic: ent: constraint failed: add m2m edge for table subject_students: UNIQUE constraint failed: subject_students.id [recovered]
	panic: ent: constraint failed: add m2m edge for table subject_students: UNIQUE constraint failed: subject_students.id